sensor_msgs/PointCloud2 feature 
std_msgs/Header header 
int32 idx
float64[] q_itoj
float64[] t_itoj
